sql-plugin-api/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

```diff
@@ -149,17 +135,14 @@ object ShimLoader extends Logging {
         // fast path
         logInfo(s"findURLClassLoader found a URLClassLoader $urlCl")
         Option(urlCl)
-      case replCl if replCl.getClass.getName == "org.apache.spark.repl.ExecutorClassLoader" ||
-          replCl.getClass.getName == "org.apache.spark.executor.ExecutorClassLoader" =>
-        // Spark 3.5.0 changed the package of ExecutorClassLoader so we check for it being
-        // either old package name or new one.
+      case replCl if replCl.getClass.getName == "org.apache.spark.repl.ExecutorClassLoader" =>
```

Comment:
is this change on purpose?

Reply:
this is a merge conflict resolution error. thanks for the catch!

Should be caught by #9504 when it goes in